gitmergesgraph.py

```diff
@@ -72,7 +72,7 @@ def load_graph(release_queue, maxNodes):
                 # connect this node with the previous one from of the same lane
                 previous_node = get_previous_node_lane(prev_node_lane, lane)
 
-            if previous_node == None:
+            if previous_node is None:
```

Comment:
@iarspider , this script does not run in IB/PRs, so please update the shibang to use python3